src/main/java/roomescape/Waiting/WaitingService.java

```diff
@@ -0,0 +1,61 @@
+package roomescape.Waiting;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import roomescape.member.LoginMember;
+import roomescape.member.Member;
+import roomescape.member.MemberRepository;
+import roomescape.reservation.MyReservationResponse;
+import roomescape.theme.Theme;
+import roomescape.theme.ThemeRepository;
+import roomescape.time.Time;
+import roomescape.time.TimeRepository;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class WaitingService {
+
+    private final WaitingRepository waitingRepository;
+    private final MemberRepository memberRepository;
+    private final TimeRepository timeRepository;
+    private final ThemeRepository themeRepository;
+
+    public WaitingResponse save(LoginMember loginMember, WaitingRequest waitingRequest) {
+
+        Member member = memberRepository.findById(loginMember.getId()).get();
```

Comment:
```suggestion
        Member member = memberRepository.findById(loginMember.getId())
                .orElseThrow(() -> new IllegalArgumentException("회원이 존재하지 않습니다."));
```
Optional 객체를 바로 꺼내올 경우, 인텔리제이에서 값의 존재 여부를 확인하지 않고 가져오는 것에 대한 알림 창이 뜨는걸 알 수 있습니다. 반환받은 Optional이 비어있을 경우를 대비해 조건문을 추가할 수 있을 것 같으며 코드를 더욱 간소화하고 직관적으로 나타내기 위해서는 `.orElseThrow()`를 쓸 수 있을 것 같습니다.

스터디에서 언급했던 자료를 확인해보면 좋을 것 같아요!
https://www.daleseo.com/java8-optional-effective/